### DISCUSSION TITLE
Cookie exceptions

### DISCUSSION THREAD

**Discussion by AnonBlankzy** [Slion/Fulguris#645] [Ideas]
I really like Fulguris Browser but the only reason I don't use it for my main browser is that there seems to be no way to delete all cookies while whitelisting the ones I want to keep. DDG browser does the best job integrating this feature with their fireproofing option. Anyway,  would love it if Fulguris implemented this

**Comment by Slion** [comment]
Good point there is room for improvement there.

**Reply by alibaubau** [reply]
For sure it would be a big improvement! 
So I could use a search engine with configured parameters, even if cookies are deleted when closing the app.
It would fit easily in the apps domain option alltogether with the existing menu for third party cookies...
Anyway, this app is great!
